Gemfile

```diff
@@ -1,16 +1,19 @@
 source 'https://rubygems.org'
 gemspec
 
-gem 'debugger',      '~> 1.6.6'
+gem 'byebug', '~> 3.5.1' if RUBY_VERSION > '2'
 gem 'yard',          '~> 0.8.5'
 gem 'rake',          '~> 0.9'
 gem 'awesome_print', '~> 1.1.0'
 gem 'redcarpet'
 
-gem 'representors', git: 'https://www.github.com/mdsol/crichton-representors.git', tag: 'v0.0.1.pre'
+gem 'representors', git: 'https://www.github.com/mdsol-share/representors.git', branch: '0-0-stable'
 
 group :development, :test do
   gem 'pry'
+  #TODO replace both crichton and crichton_test_service with references to stable branches when ready.
+  gem 'moya', git: 'https://www.github.com/mdsol/moya.git', branch: 'develop'
+  gem 'crichton', git: 'https://www.github.com/mdsol-share/crichton.git', branch: 'develop'
```

Comment:
These should also be linked to a sha or a tag.  If those don't exist, I don't see the point of releasing a critical tool that is built off gems that are shifting sand or not brave enough to tag themselves as "something that works".